AutomaticQC/SpikeClassifiers/imosSpikeClassifierNonBurstSavGolOTSU.m

```diff
@@ -51,47 +51,49 @@
 %
 
 if nargin == 1
-    window = 5; %minimum filtering
+    window = 3; %minimum filtering
     pdeg = 2; %quadratic polynomial
-    nbins = 256; % default otsu bins
+    nbins = 100; % default otsu bins
     oscale = 1; %arbitrary scale used to reduce the otsu threshold
 elseif nargin == 2
     pdeg = 2; %quadratic polynomial
-    nbins = 256; % default bins in despiking1
+    nbins = 100; % default bins in despiking1
     oscale = 1; %arbitrary scale used to reduce the otsu threshold
 elseif nargin == 3
-    nbins = 256; % default bins in despiking1
+    nbins = 100; % default bins in despiking1
     oscale = 1; %arbitrary scale used to reduce the otsu threshold
```

Comment:
This changes the values back to what was used in the original version of the code. It's also better because the defaults now are towards the limit range of the values.